extensions/amp-carousel/amp-carousel.md

```diff
@@ -78,6 +78,10 @@ In the following example, we have a carousel of three images with preview button
 </amp-iframe>
 </div>
 
+### Adding toggle controls for autoplay
+
+Setting a method for the `on` attribute on an element to `tap:carousel-id.toggleAutoPlay(toggleOn=val)` will, on user tap or click, toggle the autoplay status for the carousel with the "carousel-id" ID.
```

Comment:
we normally document them here: https://github.com/ampproject/amphtml/blob/master/spec/amp-actions-and-events.md

Reply:
Done